contracts/nns/contract.go

```diff
@@ -629,7 +629,8 @@ func Resolve(name string, typ recordtype.Type) []string {
 	}
 
 	ctx := storage.GetReadOnlyContext()
-	return resolve(ctx, nil, name, typ, 2)
+	res := []string{}
```

Comment:
the compiler understands inline nil but not empty slice?

Reply:
hm, yes, why not just `[]string{}` as an arg?

Reply:
Fixed in #420.